### PR TITLE
Logistic Regression -- Updated formatting

### PR DESCRIPTION
**Pull request by trgardos**

### CODE REVIEW

**Comment by trgardos** [comment]
@sladenheim -- it's only 38 slides. i'll think about if there's more content to add, but otherwise we could consider starting the next lecture early? I think "regularization" also might be short.

File: ds701_book/18-Regression-II-Logistic.qmd [hunk 1]
```diff
@@ -288,53 +432,70 @@ So, in summary, we have:
...
-The standard package will give us a correlation coefficient (a $\beta_i$) for each independent variable (feature).
+The standard package will give us a correlation coefficient (a $\beta_i$) for each
+independent variable (feature).
 
-If we want to include a constant (ie, $\alpha$) we need to add a column of 1s (just like in linear regression).
+::: {.content-visible when-profile="slides"}
+## Logistic Regression in Practice
+:::
+
+If we want to include a constant (ie, $\alpha$) we need to add a column of 1s (just
```
Comment (sladenheim) [inline ds701_book/18-Regression-II-Logistic.qmd:447]: ie should be i.e.,

File: ds701_book/18-Regression-II-Logistic.qmd [hunk 2]
```diff
@@ -288,53 +432,70 @@ So, in summary, we have:
...
 
 ## Using the Model
 
-Note that by fitting a model to the data, we can make predictions for inputs that were never seen in the data.  
+Note that by fitting a model to the data, we can make predictions for inputs that
+were never seen in the data.  
 
-Furthermore, we can make a prediction of a probability for cases where we don't have enough data to estimate the probability directly -- e.g, for specific parameter values.
+Furthermore, we can make a prediction of a probability for cases where we don't
+have enough data to estimate the probability directly -- e.g, for specific
```
Comment (sladenheim) [inline ds701_book/18-Regression-II-Logistic.qmd:483]: e.g., (missing a period)

**Review by sladenheim — COMMENTED** [review]
A few minor typos.

One comment to consider is should we change the \alpha + \beta x coefficients to be \beta_0 and \beta_1 to match the previous lecture.

**Comment by trgardos** [comment]
> @sladenheim -- it's only 38 slides. i'll think about if there's more content to add, but otherwise we could consider starting the next lecture early? I think "regularization" also might be short.

For example I could add another example or two.

File: ds701_book/18-Regression-II-Logistic.qmd [hunk 3]
```diff
@@ -170,29 +257,44 @@ ax.set_ylabel('Log-Odds', fontsize = 14)
 ax.set_title('Logit Function: $\log (p/1-p)$', fontsize = 16);
 ```
 
-So, logistic regression does the following: it does a linear regression of $\alpha + \beta x$ against $\log p/(1-p)$.
+::: {.content-visible when-profile="slides"}
+## Odds and Log-Odds
+:::
+
+So, logistic regression does the following: it does a linear regression of
+$\alpha + \beta x$ against $\log p/(1-p)$.
```
[the remark this answers is not in the document]
Comment (trgardos) [reply ds701_book/18-Regression-II-Logistic.qmd:265]: done

File: ds701_book/18-Regression-II-Logistic.qmd (hunk 2 above)
Comment (trgardos) [reply ds701_book/18-Regression-II-Logistic.qmd:483]: fixed

File: ds701_book/18-Regression-II-Logistic.qmd (hunk 1 above)
Comment (trgardos) [reply ds701_book/18-Regression-II-Logistic.qmd:447]: done

**Review by sladenheim — APPROVED** [review]

**Pull request merged**